pkgs/racket-doc/scribblings/reference/logging.scrbl

```diff
@@ -132,7 +132,9 @@ Reports @racket[logger]'s default topic, if any.}
 A @tech{parameter} that determines the @tech{current logger}.}
 
 
-@defform[(define-logger id)]{
+@defform[(define-logger id maybe-parent)
+         #:grammar ([maybe-parent (code:line) (code:line #:parent parent-expr)])
+         #:contracts ([parent-expr logger?])]{
```

Comment:
Can the contract be `(or/c logger? #false)` instead?

Reply:
Good catch. I’ve pushed an updated version that allows `#f`.